### ISSUE TITLE
provide one example code for depthwise convolution

### ISSUE THREAD

**Issue by yuqj1991** [oneapi-src/oneDNN#926]
Hi, guys
could you please one example code for depthwise convolution

thanks

**Comment by dzarukin** [comment]
Hi @yuqj1991, thanks for the question.
I'm very sorry, I believed I pushed the "Comment" button, but seems I didn't.

Unfortunately, we don't have such example in master, but I prepared necessary changes and opened a branch where existing convolution example [has group support](https://github.com/dzarukin/oneDNN/blob/dzarukin/convolution_with_groups_example/examples/primitives/convolution.cpp).
Diff with the original example is available [here](https://github.com/dzarukin/oneDNN/commit/1bd7d01be3b058ec6ecab58f9591f8a96d71a362).
Most of times, depthwise convolution is the one which has number of groups equal to input channels equal to output channels. From API perspective we ask to pass {G, OC/G, IC/G, KH, KW} as weights dimensions.

Hope it helps. In case of questions feel free to post them, I'll make my best to bush the button immediately. Sorry again about it.
